torch_xla/csrc/runtime/openxla_exported_symbols.lds

```diff
@@ -1,4 +1,3 @@
-*tensorflow*
 *absl*
```

Comment:
I have a feeling that a lot of these won't be needed anymore, but we can clean it up later

Reply:
sure, thanks for pointing, will add in the list too, will have follow-up PR to update doc and test to delete these lds files too

Reply:
This file has been deleted in https://github.com/pytorch/xla/pull/5275